src/DotNetty.Buffers/AbstractByteBuffer.cs

```diff
@@ -302,6 +303,12 @@ public virtual long GetLong(int index)
 
         public virtual char GetChar(int index) => Convert.ToChar(this.GetShort(index));
 
+        public virtual float GetFloat(int index)
+        {
+            int raw = this.GetInt(index);
+            return Unsafe.As<int, float>(ref raw);
```

Comment:
Can you just write a simple unsafe cast here? e.g. [http://stackoverflow.com/a/27238358](http://stackoverflow.com/a/27238358)